api/common/constants.py

```diff
@@ -0,0 +1,4 @@
+DATA_SAVED_SUCCESSFULLY = "Data saved successfully"
+INVALID_DATA = "Invalid Data"
+SUCCESS = "SUCCESS"
```

Comment:
```suggestion
SUCCESS = "Success"
```
Also, Success is meaningless, write messages more specific.